dk_sdfe/dk_sdfe_README.txt

```diff
@@ -1,36 +1,25 @@
 # dk_sdfe_README.txt
 
 The files in this section result from the conversion of datasets originating
-from [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
```

Comment:
the 2 trailing spaces (here and in below occurences) are actually needed for nice displaythe 2 trailing spaces (here and in below occurences) are actually needed for nice display

Reply:
Ah yes, I forgot about that, thanks. The one place I don't want the editor to clean trailing whitespace up. I've added similar whitespace to the SDFI readme so that renders nicely as well.